CHANGELOG.md

```diff
@@ -1,13 +1,23 @@
 # Changelog
 
-## 7.0.0 (2022-04-23)
+## 7.0.0 (2022-05-30)
 
 This version contains **breaking** changes: some errors generated by the new version will no longer group in the Bugnsag dashboard with equivalent errors generated by older versions. This is due to the introduction of a new exception interceptor that intercepts the C# exceptions objects, rather than parsing the information from Unity logs.
```

Comment:
```suggestion
This version contains **breaking** changes: some errors generated by the new version will no longer group in the Bugnsag dashboard with equivalent errors generated by older versions. This is due to the introduction of a new exception interceptor that gets passed the C# exception objects, rather than having to parse the information from Unity logs.
```